faculty-affiliations.csv

```diff
@@ -3910,6 +3909,17 @@ Victor Shoup , New York University
 Yevgeniy Dodis , New York University
 Yi-Jen Chiang , New York University
 Zvi M. Kedem , New York University
+Marsha J. Berger, New York University
+Joseph Bonneau, New York University
+Joan Bruna, New York University
+Joan Bruna Estrach, New York University
+Yann LeCun, New York University
```

Comment:
The vast majority of my publications is for research I do with my students and postdocs at NYU. Only two or three of my publications in the last 4 years are for work primarily done at Facebook.
I maintain a research group at NYU with 4 PhD students and 2 postdocs, I teach a course every year, and I participate in the activities of the department.

Reply:
Hi - if you are in fact full-time at NYU (per @deniszorin, this is the case), then you are eligible for inclusion. Please confirm. Thanks.

Reply:
My list is based on NYU faculty handbook description of full-time faculty membership (as applied to our department: tenured+ tenure-track + full-time contract), please clarify if this is what you have in mind.

Reply:
Emery: the point is that my personal research contributions must be attributed to one of my employers. The question is which one. I am a tenured full professor at NYU where I have a lab, students and postdocs. Almost all of my publications of the last 4 years are with my NYU group. My role at Facebook is one of research management and technical leadership, but I rarely get involved in Facebook projects sufficiently to have my name on any paper (with rare exceptions). Not attributing my publications to NYU would be very unfair to me and my collaborators. Thanks.

Reply:
By full-time, I mean that the faculty member is working full-time (equivalent of a 9-month appointment, so >= 75%) at the given institution.

Reply:
(Crossed messages.)

Affiliation as listed on publications is not the criterion. It's official, full-time affiliation at a particular institution. At this time, faculty who drop below the threshold given above are not eligible. Just as an example, we have a faculty member who has a half-time appointment here and who has listed UMass as their affiliation in their papers for years (while having been on leave, full-time employed elsewhere). That person is not listed in CSrankings.

Reply:
This has nothing to do with what institution I list on my papers and  everything to do with where and by whom the research is done and funded. 
An increasingly large number of academics in machine learning and AI have also affiliations in industry. Unless you use a sensible rule to take their academic labs' contributions into account your work will ignore a large swath of work.

Reply:
I think no formal criterion will ever capture the intuitively "right thing" in all cases, other than examining the details of the situation in each case --  I doubt this is in  @emeryberger's plans.  But with multiple affiliations, there is more to it rather than one specific case, especially as far as international universities are concerned (think France -- some of my colleagues there lost count). 

 In the spirit of what FAQ says ( "count the right beans.")   let me try to propose a constructive tweak on the current inclusion criterion, that would not require a full-time staff to manage, or AI to extract data from dblp  [And whatever the outcome, I think the overall idea is far more useful vs US News and others, as far as CS is concerned]
I have some ideas -- I'll post tomorrow.   Not sure if github pull request comments is the best place for this, but why not.

Reply:
CSrankings aims to "count the right beans" and  have an  entirely transparent system.  Two main data sources are used.  One is dblp, the other (faculty lists) originally came from Brown http://cs.brown.edu/people/alexpap/faculty_dataset.html, which was created based on information on the Web, collected through mechanical Turk,  and then further corrected by whoever volunteered to fix it.  The  Brown criterion for inclusion was different from the current one.   Dblp data is entirely based on open information, the second (faculty list) was initially the same way, but the current  definition requires access to non-open data for verification.  In a non-scientific random sample of people with complicated  arrangements, which _may_ not meet the criterion,  I see most included. 

Let me make a general proposal: use wikipedia verification approach accepting/rejecting any changes https://en.wikipedia.org/wiki/Wikipedia:Verifiability I.e., any change request comes with a reference link meeting a standard (I propose one below). Imperfect, but works reasonably well on much larger scale, and keeps the whole system transparent (i.e., anyone with internet access can check everything). This implies that  the criterion itself should be formulated in a way that it is verifiable by open  information on the Web.

Consider the current criterion: full time, tenure-track, can advise CS PhD students.  With my administrative hat on, "full-time" is an ambiguous term.  The elaboration in this thread,  "working 75% of the year" remains unclear, and in any interpretation is not verifiable from open sources, for example: 
-- literal interpretation: 75% of time spend doing actual work.  impossible to measure  (and certainly not verifiable).
-- salary-based:  for what fraction of time a faculty member is paid. Precise, but not verifiable from open sources  and unstable:  sabbaticals, leave,  consulting  may result in salary reduction for a fixed period. 
-- contract-based: what the appointment letter says. This is also precise, but not verifiable. However, it has a high correlation with  the title the listings on department web pages.

Proposed criterion:  A faculty member is included if he or she holds a regular faculty position at the university (for US: Prof./Associate Prof./Assistant Prof., if other countries are included this needs expansion), or an equivalent title  (e.g., Principal Research Scientist at MIT).  and can advise CS PhD students. (Whether to restrict this to tenured faculty or include research faculty may be another debate).  Acceptable verification:  a link to a faculty list with titles, maintained by a CS department (not a personal page),  showing the person's title.  In the case of unconventional titles, a link to a university policy indicating that the title is substantially similar to a faculty appointment  (can advise PhD students/apply for grants).  For non-CS faculty advising CS PhD students, a link to at least one CS thesis showing that this person was an advisor. 
This criterion will minimize corrections that need to be made to the Brown  database, as it seems to be more consistent with the way it was constructed.  Note that it does not preclude multiple affiliations if institutions allow someone to be regular faculty in two places. 

Multiple affiliations.  Some people have genuine multiple affiliations. It is less usual in the US (but does happen: e.g., https://people.eecs.berkeley.edu/~yelick/).  These are more common and often complicated in other countries. Fractions of an appointment cannot  generally be obtained from open sources. The only realistic way I see is to  assign, by default, all papers to the first listed appointment.  There should be a correction mechanism.  I propose to allow a faculty member with multiple affiliations,  to define what is his/her primary affiliation for research purposes, by creating an open reference for this, e.g., by publishing, in any more or less permanent way,  a statement describing what institution most of his research is connected to.   A better approach, requiring changes to the software, is to allow a faculty member to specify percentages of commitment (using a published statement).

Reply:
Thanks for the thoughtful proposal. Let's move this to an issue so it can get the attention and visibility it deserves.

Reply:
In the interim and in the interest of forward progress until this issue is resolved, please remove @ylecun from the commit and I can merge your PR. Thanks.